// [[Reason: undefined]] ////

//// [tests/cases/conformance/parser/ecmascript5/IndexSignatures/parserIndexSignature5.ts] ////

===================================================================
--- TSC	declarations
+++ DTE	declarations
@@ -1,5 +1,6 @@
 
 
 //// [/.src/parserIndexSignature5.d.ts]
 interface I {
+    [a]: any;
 }
